doc/storage.md

```diff
@@ -8,13 +8,16 @@ discourse: 7735
 :maxdepth: 1
 
 About storage </explanation/storage>
-Create a storage pool <howto/storage_create_pool>
+Create a pool <howto/storage_create_pool>
+Configure a pool <howto/storage_configure_pool>
+List pools <howto/storage_list_pools>
+Resize a pool <howto/storage_resize_pool>
 Create an instance in a pool <howto/storage_create_instance>
-Add a storage volume <howto/storage_add_volume>
-Configure storage settings <howto/storage_configure>
-List pools and volumes <howto/storage_list>
-Move or copy volumes <howto/storage_move>
-Back up storage volumes <howto/storage_backup>
-Resize storage <howto/storage_resize>
+Create a volume <howto/storage_create_volume>
+Configure a volume <howto/storage_configure_volume>
+List volumes <howto/storage_list_volumes>
```

Comment:
How would you feel about using "View volumes" to cover both list and show here?

Reply:
Yes, that's a good alternative!

Reply:
Thanks. That will also provide a place to add `lxc storage info` and `lxc storage volume info` in the future if we want to.